docs/conf/conf.py

```diff
@@ -45,6 +45,12 @@
 
 html_theme = "pydata_sphinx_theme"
 
+# -- sphinxcontrib-mermaid options ---------------------------------------------
+# https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest/
+
+mermaid_version = "11.5.0"
+mermaid_include_elk = "0.1.7"
```

Comment:
```suggestion
mermaid_include_elk = "0.1.7"
mermaid_version = "11.5.0"
```
nit: Do we want to maintain the alphabetical order? Or do we prefer to specify the mermaid "version" first